src/createAdder.js

```diff
@@ -22,7 +22,11 @@
  * @return {number}
  */
 function createAdder(initialValue = 0) {
-  // write code here
+  let res = initialValue;
+  return function(...number) {
+    res += number.reduce((sum, index) => sum + index, 0);
```

Comment:
you can write like this
```suggestion
   return (...numbers) => res += numbers.reduce((sum, index) => sum + index, 0);
```